packages/react-native/sdks/hermes-engine/utils/build-hermes-xcode.sh

```diff
@@ -16,6 +16,9 @@ function get_platform_copy_destination {
     if [[ $1 == "macosx" ]]; then
       echo "macosx"
       return
+    elif [[ $1 == "xros" || $1 == "xrsimulator" ]]; then
+      echo "xros"
+      return
```

Comment:
We probably don't need the local build for Xcode. Should I remove it?

Reply:
no, we need it. Otherwise, when you work in development on your laptop, for example building RNTester, it will probably fail.